src/components/schemas/report.yaml

```diff
@@ -339,7 +339,21 @@ admin_report_out:
       example: Sent Back
       description: readable state name
 
-
+admin_report_in:
+  type: object
+  additionalProperties: false
+  required:
+   -purpose
+   -id
+  properties:
+    id:
+      $ref: './fields.yaml#/id_string'
+    purpose:
+      type: string
+      description: |
+        Indicates purpose of the reports. It can only be updated for an existing report before report is moved into  `PAYMENT_PENDING` state.
```

Comment:
Please include exact states as mentioned in the ED https://www.notion.so/fyleuniverse/Edit-Report-Name-74f8d08c6a6d4c258b8dd53532e01f3b?pvs=4#8ee454ea0f9849c1a118e7434518dc5b